test/components/developers/results_visibility_component_test.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Developers
+  class ResultsVisibilityComponentTest < ViewComponent::TestCase
+    include DevelopersHelper
+
+    setup do
+      20.times do
+        create_developer(hero: "Available", available_on: Date.today)
+      end
```

Comment:
Instead of creating 20 additional developers (for every test) we can pass in `items_per_page` to `DeveloperQuery`. Then we might only have to create an additional developer or two, which should speed these tests up quite a bit.

Reply:
Indeed, that would be much better.